### PR TITLE
This will port the invoice_lines_adjustments derived table to metadb

### PR DESCRIPTION
**Pull request by sbeltaine**
Will fix issue #559. Please note that the fund distributions data array in the adjustments data array will be extracted in a derived table called invoice_lines_adjustments_fund_distributions in a future release.

```
All queries:
- [ ] Release notes added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

**Comment by jenmawe** [comment]
@sbeltaine When I run the script, fund distributions still has [] which I take that this is still an array that hasn't been extracted. However, I'm not that familiar with invoices and can't find any examples in sandbox.

**Comment by sbeltaine** [comment]
> @sbeltaine When I run the script, fund distributions still has [] which I take that this is still an array that hasn't been extracted. However, I'm not that familiar with invoices and can't find any examples in sandbox.

@jenmawe
Thanks for reviewing this query again. We will be creating a new derived table called invoice_lines_adjustment_fund_distributions in the next release (Orchid) so we can capture the adjustment fund distributions.

**Review by adoerrer — APPROVED** [review]
Having the adjustment fund distributions in a separate derived table makes sense in a future release because this would duplicate the adjustments rows.

So this looks good, thanks!

All queries:
- [x] Release notes added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary
Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Review by jenmawe — APPROVED** [review]
All queries:
- [x] Release notes added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Review by adoerrer — APPROVED** [review]
Thanks! 
Removing the adjustments.fundDistributions array since it is not extracted makes sense as well.

All queries:
- [x] Release notes added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary
Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/invoice_lines_adjustments.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+WITH adjustments AS (
+    SELECT
+        id AS invoice_line_id,
+        jsonb_extract_path_text(adjustments.data, 'description') AS adjustment_description,
+--jsonb_extract_path_text(adjustments.data, 'fundDistributions') AS adjustment_fund_distributions,
+        jsonb_extract_path_text(adjustments.data, 'prorate') AS adjustment_prorate,
+        jsonb_extract_path_text(adjustments.data, 'relationToTotal') AS adjustment_relationToTotal,
+        jsonb_extract_path_text(adjustments.data, 'type') AS adjustment_type,
+        jsonb_extract_path_text(adjustments.data, 'value')::numeric(19,4) AS adjustment_value,
+        jsonb_extract_path_text(invl.jsonb, 'adjustmentsTotal')::numeric(12,2) AS adjustment_adjustments_total
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/invoice_lines_adjustments.sql:21]: Why are the data types of these two columns different?

File: sql_metadb/derived_tables/invoice_lines_adjustments.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+DROP TABLE IF EXISTS invoice_lines_adjustments;
+
+CREATE TABLE invoice_lines_adjustments AS
+WITH adjustments AS (
+    SELECT
+        id AS invoice_line_id,
+        jsonb_extract_path_text(adjustments.data, 'description') AS adjustment_description,
+--jsonb_extract_path_text(adjustments.data, 'fundDistributions') AS adjustment_fund_distributions,
+        jsonb_extract_path_text(adjustments.data, 'prorate') AS adjustment_prorate,
+        jsonb_extract_path_text(adjustments.data, 'relationToTotal') AS adjustment_relationToTotal,
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/invoice_lines_adjustments.sql:18]: Uppercase letters should not be used in table or column name.  Please see the contributing guidelines.  I will make the corrections here.

**Pull request merged**